Emby.Server.Implementations/HttpServer/HttpListenerHost.cs

```diff
@@ -233,8 +233,10 @@ private async Task ErrorHandler(Exception ex, IRequest httpReq, bool logExceptio
                 var statusCode = GetStatusCode(ex);
                 httpRes.StatusCode = statusCode;
 
-                httpRes.ContentType = "text/html";
-                await httpRes.WriteAsync(NormalizeExceptionMessage(ex.Message)).ConfigureAwait(false);
+                var errContent = NormalizeExceptionMessage(ex.Message);
+                httpRes.ContentType = "text/plain";
```

Comment:
https://docs.microsoft.com/en-us/dotnet/api/system.net.mime.mediatypenames.text?view=netstandard-2.1 :(

Reply:
I just did how it was done all over the place there.